pkg/resource/prunerresource.go

```diff
@@ -0,0 +1,96 @@
+package resource
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	imageregistryv1 "github.com/openshift/api/imageregistry/v1"
+	imageregistryv1clientset "github.com/openshift/client-go/imageregistry/clientset/versioned/typed/imageregistry/v1"
+	imageregistryv1listers "github.com/openshift/client-go/imageregistry/listers/imageregistry/v1"
+	"github.com/openshift/cluster-image-registry-operator/defaults"
+	"github.com/openshift/cluster-image-registry-operator/pkg/parameters"
+)
+
+var (
+	defaultPrunerSuspend                    = false
```

Comment:
ditto - does using `const` instead of `var` break the pointer requirement?

Reply:
Correct, I was unable to use a constant here.